scss/window/_layout.scss

```diff
@@ -21,19 +21,23 @@
 
     // Title bar
     .k-window-titlebar {
-        padding: $window-titlebar-padding-y $window-titlebar-padding-x;
-        min-height: 16px;
-        border-width: 0 0 1px;
-        border-style: solid;
```

Comment:
The min height is actually as big as an icon (the minimum content). Actually, I am not sure we need it since we use flexbox... We could definitely improve this bit.

Border is 1px solid by design e.g. in order to fit in the theme we must have it like this. We could "hide" it by setting `$window-titlebar-border: transparent`.